src/sdk/namespace_fs.js

```diff
@@ -1610,7 +1610,12 @@ class NamespaceFS {
                     await this.set_fs_xattr_op(fs_context, versioned_path, filter_fs_xattr(directory_stat.xattr), undefined, "w");
                 }
             }
-            await this._clear_user_xattr(fs_context, latest_version_dir_path, XATTR_USER_PREFIX);
+            try {
+                await this._clear_user_xattr(fs_context, latest_version_dir_path, XATTR_USER_PREFIX);
```

Comment:
Hey @nadavMiz why not adding this catch to the clear function? or having it with a flag, I saw it'll be useful in https://github.com/noobaa/noobaa-core/pull/8975 as well